test/specs/utils/config.js

```diff
@@ -136,8 +108,8 @@ describe('config util', () => {
 										given.thereIsAConfigLockfile,
 										() => {
 											When(
-												'the config is loaded',
-												when.theConfigIsLoaded,
+												'setConfig is called',
```

Comment:
Now we're mixing and matching `setConfig` and `getConfig`. Do we want two different paths?

Reply:
I thought this config tests are mostly based on setup rather than function, so I followed the flow.